cli/src/cli_util.rs

```diff
@@ -3387,6 +3387,7 @@ impl CliRunner {
     #[must_use]
     #[instrument(skip(self))]
     pub fn run(mut self) -> ExitCode {
+        clap_complete::CompleteEnv::with_factory(crate::commands::default_app).complete();
```

Comment:
nit: `|| self.app.clone()` is probably better. I think it can support custom commands.